hyundai_kia_connect_api/KiaUvoApiAU.py

```diff
@@ -695,6 +696,25 @@ def lock_action(
         _check_response_for_errors(response)
         return response["msgId"]
 
+    def set_windows_state(
+        self, token: Token, vehicle: Vehicle, options: WindowRequestOptions
+    ) -> str:
+        url = self.SPA_API_URL_V2 + "vehicles/" + vehicle.id + "/control/windowcurtain"
+
+        payload = {
+            "backLeft": options.back_left,
+            "backRight": options.back_right,
+            "frontLeft": options.front_left,
+            "frontRight": options.front_right,
```

Comment:
The API schema seems to theoretically support it, but I didn't test opening different combinations of windows yet - only all open, or all closed. 

The Bluelink app only has open/close all windows as well, so... I'll test that later and see how it goes. Will adjust the function signature if it's not possible to do combinations.